apps/api/src/app/notification-template/e2e/update-notification-template.e2e.ts

```diff
@@ -47,6 +47,27 @@ describe('Update notification template by id - /notification-templates/:template
     expect(change._entityId).to.eq(foundTemplate._id);
   });
 
+  it('should throw error if trigger identifier already exists', async function () {
+    const notificationTemplateService = new NotificationTemplateService(
+      session.user._id,
+      session.organization._id,
+      session.environment._id
+    );
+    const template1 = await notificationTemplateService.createTemplate();
+    const template2 = await notificationTemplateService.createTemplate();
+    const update: IUpdateNotificationTemplateDto = {
+      identifier: template1.triggers[0].identifier,
+    };
+
+    const { body } = await session.testAgent.put(`/v1/notification-templates/${template2._id}`).send(update);
+
+    expect(body.statusCode).to.equal(400);
+    expect(body.message).to.equal(
+      `Notification template with identifier ${template1.triggers[0].identifier} already exists`
+    );
+    expect(body.error).to.equal('Bad Request');
+  });
+
   it('should generate new variables on update', async function () {
```

Comment:
Could we add another assertion to the existing update endpoint tests that checks the success of a identifier update?

Reply:
Sure, I will add